plugins/com.google.cloud.tools.eclipse.jdt.launching/src/com/google/cloud/tools/eclipse/jdi/internal/connect/_IntegerArgumentImpl.java

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2016 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *     Google Inc - add support for accepting multiple connections
+ *******************************************************************************/
+
+package com.google.cloud.tools.eclipse.jdi.internal.connect;
+
+import com.sun.jdi.connect.Connector.IntegerArgument;
+
+/**
+ * A fork of {@link org.eclipse.jdi.internal.connect.IntegerArgumentImpl},
+ * required as the original has package protection.
+ */
+public class _IntegerArgumentImpl implements IntegerArgument {
```

Comment:
Making this package protected does not seem to break compilation. Do we still need to fork this code?